tests/ui/legacy_numeric_constants_unfixable.stderr

```diff
@@ -0,0 +1,83 @@
+error: importing legacy numeric constants
+  --> $DIR/legacy_numeric_constants_unfixable.rs:10:5
+   |
+LL | use std::u128 as _;
+   |     ^^^^^^^^^
+   |
+   = help: remove this import
+   = note: `-D clippy::legacy-numeric-constants` implied by `-D warnings`
+   = help: to override `-D warnings` add `#[allow(clippy::legacy_numeric_constants)]`
+
+error: importing legacy numeric constants
+  --> $DIR/legacy_numeric_constants_unfixable.rs:14:24
+   |
+LL |     pub use std::{mem, u128};
+   |                        ^^^^
+   |
+   = help: remove this import and use the associated constants on the primitive type instead
+   = note: then `u128::<CONST>` will resolve to the respective associated constant
```

Comment:
Just looking at the second part of the help message, it sounds like the user has to do more than just remove the import.

What do you think about a change like this

```suggestion
   = help: remove this import
   = note: then `u128::<CONST>` will automatically resolve to the respective associated constant
```

Reply:
Good idea. I made the change, just without "automatically", because IMO it adds verbosity without much value.